mac

```diff
@@ -92,15 +104,12 @@ gem_install_or_update() {
 
 if ! command -v brew >/dev/null; then
   fancy_echo "Installing Homebrew ..."
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-    # Add brew to the path
-    append_to_zshrc 'eval "$(/opt/homebrew/bin/brew shellenv)"'
-    append_to_zshrc '# recommended by brew doctor'
+    /bin/bash -c \
+      "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
This is also from the upstream repo. I see Garret had previously updated this. The changes below should do the right thing with HOMEBREW_PREFIX.